pkg/controllers/provisioningv2/rke2/managesystemagent/managesystemagent.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
+	"github.com/rancher/rancher/pkg/controllers/provisioningv2/rke2"
```

Comment:
I don't think this will pass the go imports validation

Reply:
Okay, fixed in both this PR and the backport